src/Resources/views/CRUD/list_outer_rows_mosaic.html.twig

```diff
@@ -30,7 +30,7 @@ This template can be customized to match your needs. You should only extends the
 
                             <div class="mosaic-inner-box-default">
                                 {% block sonata_mosaic_background %}
-                                    <img src="{{ meta.image }}" alt="" />
+                                    <img src="{{ meta.image ?: sonata_admin.getDefaultMosaicBackground() }}" alt="" />
```

Comment:
seems wrong, since `meta.image` has a default value, this means this will never render?

Reply:
Yes until `Metadata` classes are updated in core and block bundles

Reply:
But if they are updated before it will be no background at all

Reply:
The issue is that people are using those bundle independently from sonata (as that is possible). What would make more sense now, make the default configuration null, and switch the case:

```
sonata_admin.getDefaultMosaicBackground() ?: meta.image
```

Reply:
Ah yes I was not thinking about it!
Problem with your suggestion is that if image from entity is set in meta, it will be replaced by default background...
But in fact, what is the problem with simple link to file instead of hardcoded image in `Metadata` classes, is it only because after update default background images will disappear until `assets:install`?

Reply:
No idea tbh, like this we can just use it as string everywhere and not add it in every bundle

Reply:
Ah yes understood, ok I will think twice.
Suggestions are welcome...

Reply:
This way image is still hardcoded in `Metadata` classes and bundles can still be used independently.
But if image is still hardcoded in `Metadata` classes, maybe better to remove it from `Configuration` class and update template like that:
```
{% set default_mosaic_background = sonata_admin.getDefaultMosaicBackground() %}
<img src="{{ meta.isImageAvailable or not default_mosaic_background ? meta.image : default_mosaic_background }}" alt="" />
```
?